### DISCUSSION TITLE
API Key and Obsidian Plugin - Subscription option - Lack of information and documentation

### DISCUSSION THREAD

**Discussion by brigittethecat** [khoj-ai/khoj#844] [Q&A]
1. This a beginner question: I have generated an API key.  According to the information in settings, chat gpt 4-turbo preview is being used.but I didn't provide any additional api key for chat gpt.  So, what model is being used for my chats and searches?
 
2. There's a free plan, limited to 5 queries per day. However, I can't find usage statistics. I don't know how many queries I have made and if it's necessary to upgrade . Do you know where I can find those informations? What, exactly, is considered a query? That lack of information is really upsetting me.

**Comment by debanjum** [comment]
Hi @brigittethecat, 
1. The API key is for the Khoj application. You don't need to set any additional keys for the chat model being used. You get access to it automatically with your Khoj API key.
2. We should total show those numbers. Let me implement this by end of month (as part of a general UX upgrade we're doing). To address your other concerns:
   - A query is any chat message you sent to Khoj
   - If you've just made a new account, you'll be on a 14 day trial for the paid plan. This gives you uncapped messages, ability to try out other chat models etc
   - We're also working to update the pricing page to be more accurate, standard and clear. I'll include 

I'll split the issue #845 into two separate, specific issues:
1. To show `sent messages / daily messages limit` statistic
2. To update the Khoj pricing page

Let me know if that things are still unclear?

**Reply by debanjum** [reply]
Created issue to update pricing page at khoj-ai/landing-page#37. Converted #845 into issue to show usage statistics
